modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/HashIndex.java

```diff
@@ -39,9 +39,12 @@ class HashIndex {
 
     private final ConcurrentMap<Integer, RocksDbHashIndexStorage> storages = new ConcurrentHashMap<>();
 
-    HashIndex(ColumnFamily indexCf, HashIndexDescriptor descriptor) {
+    private final RocksDbMetaStorage metaStorage;
+
+    HashIndex(ColumnFamily indexCf, HashIndexDescriptor descriptor, RocksDbMetaStorage metaStorage) {
         this.indexCf = indexCf;
         this.descriptor = descriptor;
+        this.metaStorage = metaStorage;
```

Comment:
I would prefer different name, this one looks too similar to the name of other, completely unrelated, component

Reply:
I don't know what name to give, I tried "indexMetaStorage".

Reply:
`indexMetaStorage` is a lot better